CountingWords.rb

```diff
@@ -0,0 +1,6 @@
+freq = Hash.new(0)
+file = File.open("speech.txt")
+file.read.downcase.scan(/\b[a-z]{3,16}\b/){|word| freq[word] += 1}
```

Comment:
how would you rewrite this to handle punctuation? 

your script also doesn't handle words less than 3 characters? how would you rewrite this to take those into account?